### DISCUSSION TITLE
If there is no school in the GitHub Campus program, is student certification impossible?

### DISCUSSION THREAD

**Discussion by Chemuchi** [community/community#56485] [GitHub Education]
### Hi everyone,

I keep trying to get a student certification, but I'm being rejected.

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote.

You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

I keep getting rejected for these two reasons. Please don't block the ticket. All I can do more is to inquire, but the related question is to block the ticket, so there's nothing more I can do.

**Comment by samus-aran** [comment]
Hi @Chemuchi 👋 

> The school you selected does not appear to have a campus location in your country. 

Is the accurate? If so then you need to submit evidence that you are remote learner. Is the course online for example?

Alternatively if you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "Other". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
